components/suggestion/suggestion_list.jsx

```diff
@@ -192,40 +191,39 @@ export default class SuggestionList extends React.PureComponent {
     }
 
     render() {
-        if (!this.props.open || this.props.cleared) {
+        const {
+            items,
+            terms,
+            selection,
+            components,
+            open,
+            cleared,
+            renderDividers,
+            renderNoResults,
+        } = this.props;
+
+        if (!open || cleared) {
             return null;
         }
 
-        const clonedItems = cloneDeep(this.props.items);
-
-        const items = [];
-        if (clonedItems.length === 0) {
-            if (!this.props.renderNoResults) {
+        const handledItems = [];
+        if (items.length === 0) {
+            if (!renderNoResults) {
                 return null;
             }
-            items.push(this.renderNoResults());
+            handledItems.push(this.renderNoResults());
         }
 
-        const sortedItems = clonedItems.sort((itemA, itemB) => {
-            if (!itemA.type) {
-                return 1;
-            }
-            if (!itemB.type) {
-                return -1;
-            }
-            return itemA.type.localeCompare(itemB.type);
-        });
-
         let lastType;
-        for (let i = 0; i < sortedItems.length; i++) {
-            const item = sortedItems[i];
-            const term = this.props.terms[i];
-            const isSelection = term === this.props.selection;
+        for (let i = 0; i < items.length; i++) {
+            const item = handledItems[i];
```

Comment:
@michelengelen shouldn't this be 
```suggestion
            const item = items[i];
```